dev-guide/host-port-registry.md

```diff
@@ -88,6 +88,7 @@ Ports are assumed to be used on all nodes in all clusters unless otherwise speci
 | 9103  | ovn-kubernetes node kube-rbac-proxy | sdn || metrics |
 | 9105  | ovn-kubernetes node kube-rbac-proxy-ovn-metrics | sdn | 4.10 | metrics |
 | 9106  | sdn controller kube-rbac-proxy | sdn | 4.10 | control plane only, sdn only |
+| 9107  | ovn-kubernetes node | sdn | 4.12 | egressip-node-healthcheck-port, node only |
```

Comment:
To be consistent should we also add `ovn-kubernetes only` note?

Additionally, we only listen on the ovn-k management IP, should we add that to the note as well?

Reply:
Sure thing! will do

Reply:
/hold

Reply:
done.